assets/web/pin.html

```diff
@@ -1,31 +1,64 @@
-
-<div id="content" class="container">
-    <h1 class="my-4">PIN Pairing</h1>
-    <form action="" class="form d-flex flex-column align-items-center" id="form">
-        <div class="card flex-column d-flex p-4 mb-4">
-            <input type="number" placeholder="PIN" id="pin-input" class="form-control my-4">
-            <button class="btn btn-primary">Send</button>
-        </div>
-        <div class="alert alert-warning">
-            <b>Warning!</b> Make sure you have access to the client you are pairing with.<br>
-            This software can give total control to your computer, so be careful!
+<main role="main">
+  <div class="container-parent bg-light">
+    <div class="container py-3">
+      <h1>Pair a new device</h1>
+      <p class="mb-0">Allow a new client to connect to this computer by entering the PIN code given by Moonlight here.
+      </p>
+    </div>
+  </div>
+  <div class="container-parent">
+    <div class="container py-3">
+      <div class="row justify-content-center">
+        <div class="col-md-6">
+          <div class="alert alert-warning" role="alert">
+            <strong>Warning!</strong> Make sure you have access to the client you are pairing with.<br /> This software
+            can give total control to your computer, so be careful!
+          </div>
+          <form action="#" id="form">
+            <div class="mb-3">
+              <label for="input" class="form-label">PIN code</label>
+              <input type="text" class="form-control form-control-lg" id="input" name="pin" placeholder="1234"
```

Comment:
Making this a type="number" can help on mobile keyboard, since it will show only the numbers on smartphones